src/Tools/dotnet-symbol/Program.cs

```diff
@@ -263,7 +265,7 @@ internal async Task DownloadFiles()
         private Microsoft.SymbolStore.SymbolStores.SymbolStore BuildSymbolStore()
         {
             Microsoft.SymbolStore.SymbolStores.SymbolStore store = null;
-
+            HttpClient.DefaultProxy.Credentials = CredentialCache.DefaultCredentials;
```

Comment:
This change should be in the HttpSymbolStore.  I need to think about this some more.

Reply:
Ack